src/shared/components/lazyMobXTable/PaginatedDataStore.tsx

```diff
@@ -0,0 +1,140 @@
+import { action, computed, makeObservable, observable } from 'mobx';
```

Comment:
put a nice big comment explaining that this is a "facade" to trainslate the page store into the data store expected by LazyMobxTable

Reply:
Added in [44ed633](https://github.com/cBioPortal/cbioportal-frontend/pull/4590/commits/44ed633252d40c7bb047f11fb4bc7838e07e7db7)